examples/dhs-example/README.md

```diff
@@ -0,0 +1,97 @@
+This example project demonstrates where to store resources that can be deployed to a DHS instance using a user with the 
+data-hub-developer role, or the data-hub-security-admin role, or both. 
+
+This project is not meant to be deployed. It is intended only to serve as a reference for where resource files
+should be stored. Because of this, it also does not define any Data Hub artifacts, such as flows or entities.
+
+## General deployment guidelines
+
+The tasks for deploying resources to DHS will never read from the src/main/hub-internal-config directory. The 
+resources in that directory are deployed automatically within DHS. For example, if you have defined your configuration
+directories via the following property:
+
+    mlConfigPaths=src/main/hub-internal-config,src/main/ml-config,src/main/other-config
+    
+The ml-config and other-config directories will still be processed, but hub-internal-config will not be.
+
+## Deploying resources as a data-hub-security-admin user
+
+As of 5.2.0, a user with the data-hub-security-admin role is permitted to deploy roles that grant privileges that are
+inherited by the user performing the deployment. As an example of this, the 
+src/main/ml-config/security/roles/custom-role1.json file defines a new role with the "manage" privilege, which is 
+inherited by the data-hub-security-admin role. 
+
+Permitted resources can be deployed via the following task (assuming that gradle-dhs.properties defines the host and
+credentials for the DHS instance):
+
+    ./gradlew -PenvironmentName=dhs dhsDeployAsSecurityAdmin
```

Comment:
Should this be changed to hubDeployAsSecurityAdmin?

Reply:
Good catch, made the fix.